courses/machine_learning/deepdive2/feature_engineering/solutions/1_bqml_basic_feat_eng_bqml.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<p>Historically, the solution notebooks look like a completed lab notebooks, so you can delete this TODO cell and the # SOLUTION comment below.</p><p> </p><p>The goal is to have a notebook that the student can run every cell and not run into any errors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Notebook split  into two - a Lab and a Solution.  The LAB notebook has TODO's while the SOLUTION notebook does not.

Reply:
Thanks! Any reason why you kept the "# SOLUTION"s? Historically, we haven't had markers like that because the idea was that the solution notebook should look like a completed student notebook.

Reply:
Removed the # Solution from Lab01 and Lab02.

Reply:
Removed the # Solution comment and TODO's from Lab03 and Lab04.